src/main.c

```diff
@@ -49,7 +43,7 @@ static enum swupd_code external_search_main(int argc, char **argv)
 
 // TODO(castulo): remove the superseded command from the help menu by end of November 2019,
 // so it is not visible but the command must remain available in the back so we don't break users
-static struct subcmd commands[] = {
+struct subcmd main_commands[] = {
```

Comment:
I realized that now. This should remain static.

Reply:
No. static means that the variable isn't going to be visible outside of this C source file.Using it as a variable is valid and it's ok. Doing that prevents another source file to do this:

```
extern struct subcmd main_commands[];
main_commands = NEW_VALUE;
```

This would change the value of the original main_commands variable

Reply:
@otaviobp Thanks for explaining that. I know why it works...subcmd_index is inline static fucntion and it isnt considered another file.

Reply:
Fixed this in last push